src/main/java/org/opensearch/security/support/HTTPHelper.java

```diff
@@ -39,10 +39,8 @@
 public class HTTPHelper {
 
     public static AuthCredentials extractCredentials(String authorizationHeader, Logger log) {
-
         if (authorizationHeader != null) {
             if (!authorizationHeader.trim().toLowerCase().startsWith("basic ")) {
-                log.warn("No 'Basic Authorization' header, send 401 and 'WWW-Authenticate Basic'");
```

Comment:
this line was okay to be here, I think the problem was with defined order of authenticators - basic auth as first challenger and SAML as second. See https://github.com/opensearch-project/security/pull/5221#discussion_r2017353343